src/appleseed.shaders/src/maya/as_maya_leather.osl

```diff
@@ -1,244 +0,0 @@
-
```

Comment:
What's the rationale behind remove this shader?

Reply:
I don't think *leather* was in already. It's WIP. A Maya node, 3d texture, that uses Worley noise to fake leather, with some extras which i'm having a hard time replicating. Not quite ready yet, but close. It'll be the subject of the next PR together with Gaffer UI updates.